_docs/setup/index.md

```diff
@@ -1,5 +1,108 @@
 ---
-title: Setup
+title: Forseti Installation
+order: 001
 ---
 
-TODO
\ No newline at end of file
+# {{ page.title }}
+
+This guide explains how to use the Forseti installation tool.
+
+## Before you begin
+
+Prior to running the setup wizard, you will need:
+
+  - A GCP organization for which you want to deploy Forseti.
+  - Org Admin IAM role in order for the script to assign the Forseti
+  service account roles on the organization IAM policy.
+  - A GCP project dedicated to Forseti, you can reuse the same 
+  project that has Forseti 1.0 installed in it.
+  - Enable billing on the project.
+
+
+## Setting up Forseti Security
+
+The installer automatically determines setup information, generates a
+deployment template, and creates a Forseti deployment.
+
+### Activate Google Cloud Shell
+
+It's recommended to use [Cloud Shell](https://cloud.google.com/shell/docs/quickstart) to run the
+Forseti installer. This ensures you're using the latest version of Cloud SDK since it's included
+in Cloud Shell. To prepare to run the Forseti setup wizard, follow the steps below:
+
+  1. Access the [Cloud Platform Console](https://console.cloud.google.com/).
+  1. In the **Select a project** drop-down list at the top of the console, select the project where
+  you want to deploy Forseti.
+  1. On the top right of the console, click the icon to **Activate Google Cloud Shell**. The Cloud
+  Shell panel opens at the bottom of the page.
+
+### Run setup
+
+  1. Once you've started Cloud Shell, download Forseti. The installer is included.
+     Getting `2.0-dev-rc1` branch will install Forseti v2.0 Release Candidate 1.
+
+      ```bash
+      git clone -b 2.0-dev-rc1 --single-branch https://github.com/GoogleCloudPlatform/forseti-security.git
+      ```
+
+  1. Navigate to the installer directory:
+
+      ```bash
+      cd forseti-security/setup/
+      ```
+
+  1. Run the installer:
+
+     To install both client and server:
+     ```bash 
+     python installer.py
+     ```
+     To install server only:
```

Comment:
I don't know of a use-case where someone would only install one or the other at the moment. So to prevent confusion in the docs and make it super-clean let's just not document this installation option.

Reply:
Updated.